### DISCUSSION TITLE
[Training Log 2022-09-28] We have added English training data today.

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#183] [Training Logs 训练日志]
# CPM-Live Training Log (September, 28)

Time: September, 28 2022 16:00

Recorder: @zh-zheng

## Loss
- Begin: 2.07
- End: 3.07
	
## Completed Data
- $\approx$ 1,171.16GB

## Average Grad Norm
- 0.71

## Progress
- 125.75%

## Comment

The training loss increased substantially today, as we have added English training data. Don't worry! Our model is trying to learn from the new data and the loss is dropping.
